packages/kuma-gui/src/app/x/components/x-about-section/XAboutSection.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <AppAboutSection>
+    <template
+      v-for="(_, slotName) in $slots"
+      :key="slotName"
+      #[slotName]="slotProps"
+    >
+      <slot
+        :name="slotName"
+        v-bind="(slotProps)"
+      />
+    </template>
+  </AppAboutSection>
+</template>
+
+<script setup lang="ts">
+import { AppAboutSection } from '@kong-ui-public/app-layout'
+</script>
+
+<style lang="scss" scoped>
+:deep(.about-section-content) {
+  .definition-card {
+    align-self: center;
+
+    &-title {
+      font-size: $kui-font-size-20;
+    }
+
+    &-container {
+      gap: $kui-space-20;
+      font-size: $kui-font-size-20;
+      font-weight: normal;
+    }
+
+    .x-layout.separated {
+      align-items: center;
+    }
+  }
+}
```

Comment:
These are the bits I'm mostly concerned about. I believe (but not totally sure) these are all overwrites, meaning we potentially fall out of sync with upstream, and I think the reasoning behind flipping this component back in is to prevent us from falling out of sync with upstream?

We had a little chat offline about this, specifically we noted that the important things that AppAboutSection gives us is a font-size and a font-color, yet here we overwrite a font-size. That doesn't make sense to me as we then diverge from upstream.

It would be great if we could find an option that meant we can use AppAboutSection without any overwrites at all. I don't get why we need overwrites if nobody else using the same component has overwrites.

Reply:
These overwrites are not for the upstream component, but for our internal `DefinitionCard`. There is no "standard" for the content (`#body` slot of `AppAboutSection`) and we could add anything in there. This styling just makes sure that we are (for now) aligned with how others are using this component.

Reply:
Oh I see! 🤦 Gotcha I missed that it was not really styling for this component but defined in this component, thanks!

Reply:
You shouldn't need the `x-layout.separated` here anymore a similar rule was added just before you put this PR up I think https://github.com/kumahq/kuma-gui/blob/698c2a75cd3a46f4cfb4363f8199dab77f5438b3/packages/kuma-gui/src/app/x/components/x-layout/XLayout.vue#L27

Actually, thats made me think of something else 🤔

Reply:
Ah, thanks 👍 I've removed it 🙂